### PR TITLE
Dupe check full ssns 180352601

### PR DESCRIPTION
**Pull request by bytheway875**

### CODE REVIEW

File: spec/models/ctc_intake_spec.rb [hunk 1]
```diff
@@ -219,7 +219,73 @@
...
+    end
+
+    context "when neither phone number nor email are present" do
+      let(:intake) { create :ctc_intake, email_address: nil, sms_phone_number: nil }
+      it "responds with an empty ActiveRecord relation" do
+        expect(intake.duplicates).to eq described_class.none
+      end
+    end
+  end
+>>>>>>> Add tests for duplicate functionality
```
Comment (embarnard) [inline spec/models/ctc_intake_spec.rb]: we should remove this
